src/app/components/turnos/gestor-agendas/pipes/botonesAgenda.pipe.ts

```diff
@@ -0,0 +1,125 @@
+import { Auth } from '@andes/auth';
+import { Pipe, PipeTransform } from '@angular/core';
+import { IAgenda } from '../../../../interfaces/turnos/IAgenda';
+
+@Pipe({
+    name: 'botonesAgenda'
+})
+export class BotonesAgendaPipe implements PipeTransform {
+    constructor(private auth: Auth) { }
+    transform(agenda: IAgenda): any {
+        const botones = {
+            editar: false,
+            suspender: false,
+            reasignar: false,
+            clonar: false,
+            sobreturno: false,
+            revision: false,
+            cargaMasiva: false
+        };
+
+        const puedeEditar = this.auth.getPermissions('turnos:agenda:puedeEditar:').length > 0;
```

Comment:
La forma correcta de validar estos persimos sería:
```suggestion
        const puedeEditar = this.auth.check('turnos:agenda:puedeEditar');
```